.github/workflows/format-command.yml

```diff
@@ -18,22 +18,30 @@ jobs:
 
       # Install black and isort
       - name: Install black, isort and flynt
-        run: pip install black isort flynt
+        run: |
+          python --version 
+          pip install black isort flynt
 
       # Execute black in check mode
       - name: Black
         id: black
-        run: echo ::set-output name=format::$(black --check --quiet . || echo "true")
+        run: |
+          black --version
+          echo ::set-output name=format::$(black --check --quiet . || echo "true")
 
       # Execute isort in check mode
       - name: isort
         id: isort
-        run: echo ::set-output name=format::$(isort --check --quiet . || echo "true")
+        run: |
+          isort --version
+          echo ::set-output name=format::$(isort --check --quiet . || echo "true")
 
       # Execute isort in check mode
       - name: flynt
         id: flynt
-        run: echo ::set-output name=format::$(flynt --fail-on-change --dry-run yt/ --exclude yt/extern || echo "true")
+        run: |
+          flynt --version
```

Comment:
As of flynt v.50.0, this will fail (flynt doesn't have a `--version` option).

Reply:
Hold my coffee
https://github.com/ikamensh/flynt/pull/74

Reply:
_now_ this should work !